src/translations.js

```diff
@@ -326,6 +326,26 @@ const wakatimeCardLocales = {
     vi: "Thống Kê Wakatime",
     se: "Wakatime statistik",
   },
+  "wakatimecard.lastyear": {
+    en: "last year",
+    cn: "去年",
+  },
+  "wakatimecard.last7days": {
+    en: "last 7 days",
+    cn: "最近 7 天",
+  },
+  "wakatimecard.nouser": {
+    en: "Wakatime user not found, make sure you have a Wakatime profile",
+    cn: "未找到 Wakatime 用戶，請確保您擁有 Wakatime 個人資料",
```

Comment:
This (and the following ones) should be "zh-tw", when "cn" refers to Chinese (Simplified).

Reply:
```suggestion
    cn: "未找到 Wakatime 用户，请确保您拥有 Wakatime 个人资料",
```

Reply:
Thanks! I don't speak Chinese, so I assumed the short abbreviation was the traditional one.

Reply:
It should be fixed in [7ea55d0](https://github.com/anuraghazra/github-readme-stats/pull/2707/commits/7ea55d05ade794fe7430c8eb2e50b2f5885d9fdb).